plugin/latexlivepreview.vim

```diff
@@ -209,12 +210,12 @@ EEOOFF
     endif
 
     " Enable compilation of bibliography:
-    let l:bib_files = split(glob(b:livepreview_buf_data['root_dir'] . '/**/*.bib'))     " TODO: fails if unused bibfiles
+    let l:bib_files = split(globpath(b:livepreview_buf_data['root_dir'], '**/*.bib', 1), "\n")  " TODO: fails if unused bibfiles
     if len(l:bib_files) > 0
         for bib_file in l:bib_files
             let bib_fn = fnamemodify(bib_file, ':t')
             call writefile(readfile(bib_file),
-                        \ l:tmp_root_dir . '/' . bib_fn)                                " TODO: may fail if same bibfile names in different dirs
+                        \ l:tmp_root_dir_raw . '/' . bib_fn)                            " TODO: may fail if same bibfile names in different dirs
```

Comment:
Are you sure this change is necessary?

Reply:
I am pretty sure. I tried with the `l:tmp_root_dir` variable, but it did not work. AFAIK it is because `l:tmp_root_dir` contains the path escaped (with '\\' before every space) and `writefile` does not like that. So the *raw* version is without escaping.